controllers/drcluster_controller_test.go

```diff
@@ -438,6 +438,9 @@ var _ = Describe("DRClusterController", func() {
 				By("changing the S3Profile in ramen config to an invalid value")
 				newS3Profiles := s3Profiles[0:]
 				s3Profiles[5].S3Bucket = bucketNameFail
+				Expect(k8sClient.Get(context.TODO(),
+					types.NamespacedName{Name: controllers.HubOperatorConfigMapName, Namespace: ramenNamespace},
+					configMap)).To(Succeed())
```

Comment:
Not clear what this line tests. If we cannot use more clear code maybe add a comment?

Reply:
@nirs , you are right, it is not clear. The next line (s3ProfilesStore(newS3Profiles)) is changing ConfigMap. Before doing it, it is necessary to get the latest ConfigMap (k8sClient.Get is reloading it). 

We didn't need it before adding label in this PR. Nobody changed ConfigMap, only read from it.

Reply:
In order to make code clearer, I moved the reloading of the ConfigMap into rameConfig_test file.